lib/open_project/text_formatting.rb

```diff
@@ -418,5 +425,19 @@ def replace_toc(text, headings)
         end
       end
     end
+
+    #
+    # displays the current url plus an optional anchor
+    #
+    def full_url(anchor_name = '')
+      return false if current_request.nil?
```

Comment:
````
> false.to_s
 => "false" 
````

I don't know if returning `false` is a good idea for a method that usually returns a string. Especially since this method is not always used with that in mind (and this special behaviour is undocumented).

I'd personally rather return `nil` here...